src/tools/tidy/src/issues.txt

```diff
@@ -3850,6 +3848,8 @@ ui/static/issue-18118.rs
 ui/static/issue-24446.rs
 ui/static/issue-24843.rs
 ui/static/issue-34194.rs
+ui/static/issue-3668-non-constant-value-in-constant/issue-3668-2.rs
+ui/static/issue-3668-non-constant-value-in-constant/issue-3668.rs
```

Comment:
same request but
- tests/ui/static/non-constant-value-in-constant-issue-3668.rs
- tests/ui/static/non-constant-value-in-constant-issue-3668-2.rs

and so on.